code/datums/components/archaeology.dm

```diff
@@ -0,0 +1,96 @@
+/datum/component/archaeology
+	dupe_type = COMPONENT_DUPE_UNIQUE
+	var/list/archdrops
+	var/prob2drop
+	var/dug
+
+/datum/component/archaeology/Initialize(_prob2drop, list/_archdrops = list())
+	prob2drop = Clamp(_prob2drop, 0, 100)
+	archdrops = _archdrops
+	if(isopenturf(parent))
```

Comment:
Remove this check, the beauty of components is they can work on anything (generally)